test/conformance/exp_command_buffer/commands.cpp

```diff
@@ -204,3 +204,14 @@ TEST_P(urCommandBufferAppendKernelLaunchExpTest, Basic) {
         ASSERT_EQ(result, ptrZ[i]);
     }
 }
+
+TEST_P(urCommandBufferAppendKernelLaunchExpTest, FinalizeTwice) {
```

Comment:
For native CPU the kernel tests don't work, so think this will need to be added to the match file next to the test above https://github.com/oneapi-src/unified-runtime/blob/4fb1a847b39e2cd2c5efac952e72ec3cf9dd8ddd/test/conformance/exp_command_buffer/exp_command_buffer_adapter_native_cpu.match#L7

(Bonus points if you want to look into why that's happening, but it's not a change for this PR)

Reply:
Good shout. Have updated match file